server/src/application/application.controller.ts

```diff
@@ -120,4 +122,35 @@ export class ApplicationController {
   ) {
     return this.applicationService.updateBroaderReviewScore(user, applicationId, scoreId, scoreDto);
   }
+
+  // Workshop Score Section
+  @Get('/:applicationId/workshop')
+  async getWorkshopScores(@Param('applicationId') applicationId: string) {
+    return this.applicationService.getWorkshopScores(applicationId);
+  }
+
+  @Post('/:applicationId/workshop')
+  async createWorkshopScore(
+    @Body() scoreDto: WorkshopScoreDto,
+    @GetUser() user: User,
+    @Param('applicationId') applicationId: string
+  ) {
+    if (!user.isAdmin) {
```

Comment:
No need to add this! Use the decorator that's already used! @Roles(UserRoles.ADMIN)